Sources_v3/Database/DTOs/ChannelDTO.swift

```diff
@@ -150,7 +159,8 @@ extension ChannelModel {
     /// Create a ChannelModel struct from its DTO
     static func create(fromDTO dto: ChannelDTO) -> ChannelModel {
         let members = dto.members.map { MemberModel<ExtraData.User>.create(fromDTO: $0) }
-        
+        let typingMembers = dto.typingMembers.map { MemberModel<ExtraData.User>.create(fromDTO: $0) }
```

Comment:
Maybe you can just use `$0.asModel()`?

Reply:
fixed in aed02f9263fc5863b5ae627e5303ffcfb94e08bd